tests/Unit/Domain/Speaker/SpeakerProfileTest.php

```diff
@@ -273,6 +273,40 @@ public function getTwitterReturnsTwitterIfPropertyIsNotHidden()
         $this->assertSame($twitter, $profile->getTwitter());
     }
 
+    /**
+     * @test
+     */
+    public function getJoindInUsernameThrowsNotAllowedExceptionIfPropertyIsHidden()
+    {
+        $hiddenProperties = [
+            'joindin_username',
+        ];
+
+        $speaker = $this->createUserMock();
+
+        $profile = new SpeakerProfile($speaker, $hiddenProperties);
+
+        $this->expectException(NotAllowedException::class);
+
+        $profile->getJoindInUsername();
+    }
+
+    /**
+     * @test
+     */
+    public function getJoindInUsernameReturnsJoindInUsernameIfPropertyIsNotHidden()
+    {
+        $joindin_username = $this->faker()->userName;
```

Comment:
Can we use `camelCase` here instead of `snake_case`?

Reply:
Yep, `camelCase` please